README.md

```diff
@@ -1 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Can you add a commented TODO, something like.

TODO: Add project description and simpifiy Biolerplate description.

I don't think it's a good idea for the generic description to be the first thing companies see when they look at this repo.